src/main/java/com/scaleset/cfbuilder/ec2/metadata/CFNCommand.java

```diff
@@ -0,0 +1,46 @@
+package com.scaleset.cfbuilder.ec2.metadata;
```

Comment:
We probably should have discussion on our package structure and class naming. 
I'm not sure if these classes should not have the prefix `CFN` and instead should be in a package called `cloudformation` instead of `metadata` to be more in line with the existing other classes.

I think it's fine for now, this is another thing for future versions.

Reply:
Agree, it's a little messy.